core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/get/GetTaskRequest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.action.admin.cluster.node.tasks.get;
+
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.support.tasks.BaseTasksRequest;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+
+import static org.elasticsearch.action.ValidateActions.addValidationError;
+import static org.elasticsearch.common.util.CollectionUtils.isEmpty;
+
+/**
+ * A request to get node tasks
+ */
+public class GetTaskRequest extends BaseTasksRequest<GetTaskRequest> {
```

Comment:
This might be really confusing. We don't support any parameters that are provided by `BaseTasksRequest<GetTaskRequest>`  so the only reason to use it would be to continue using TransportTasksAction which is overkill here anyway since we know the node that we need to get the task from. So, we can base this directly on TransportNodesAction instead.